va/va_dec_av1.h

```diff
@@ -365,8 +365,8 @@ typedef struct  _VADecPictureParameterBufferAV1
     uint8_t                 tile_rows;
 
     /* specifes the width/height of a tile minus 1 in units of superblocks */
-    uint16_t                width_in_sbs_minus_1[63];
-    uint16_t                height_in_sbs_minus_1[63];
+    uint16_t                width_in_sbs_minus_1[64];
```

Comment:
it will break backward compatibility

Reply:
Should I decrease this by 1? https://github.com/intel/libva/blob/master/va/va_dec_av1.h#L650

Reply:
This won't work and still breaks backward compatibility. You can not add fields or increase size of arrays in the middle of the structure. You need to add fields in the very end taking out space from va_reserved. 

Why exactly do we need this change?